docker-compose.yml

```diff
@@ -24,7 +24,7 @@ services:
       - log_volume:/dockstore_logs
       - ./config/web.yml:/home/web.yml
       - ./config/init_webservice.sh:/home/init_webservice.sh
-      - ${GITHUB_APP_PRIVATE_KEY_FILE}:/home/dockstore_github_app_private_key.pem
```

Comment:
Curious, what was the reason for making the right side (in the container) configurable? 
I can kinda justify the left side being configurable (if we had multiple private keys around because we were migrating from one key to another) but why the right side?

Reply:
Curious too. :)

Reply:
Isn't it this PR that is making it's value in `templates/web.yml.template`  configurable? 
It was previously hard coded to `/home/dockstore_github_app_private_key.pem`

Reply:
Yeah it is. The github key can't be in the `/home` directory for fargate because if we mount a bind mount at the `/home` directory, it overwrites the JAR that's also in `/home` in the `quay.io/dockstore/dockstore-webservice` image, which is why I made the the value in `web.yml` configurable. 

Would it be more preferable to hard code the path in `web.yml` to something else that also works with Fargate? Like `/home/github-key/dockstore_github_app_private_key.pem`?

Reply:
Oh I see, so the implication is that Fargate bind mounts need to replace a whole directory and cannot replace just one file.

Yeah, I think  `/home/github-key/dockstore_github_app_private_key.pem` would be conceptually simpler.